pyvlx/pyvlx.py

```diff
@@ -68,6 +68,7 @@ async def send_frame(self, frame):
 
     async def disconnect(self):
         """Disconnect from KLF 200."""
+        await house_status_monitor_disable(pyvlx=self)
```

Comment:
I'm not a big fan of comments in source code. But this line is worth one....

Reply:
...added!